install/deploy_final_non-compute_node.md

```diff
@@ -84,6 +84,25 @@ The steps in this section load hand-off data before a later procedure reboots th
     2021/02/02 14:05:15 Successfully uploaded SLS Input File.
     ```
 
+1. (`pit#`) Upload NCN boot artifacts into S3.
+
+    ```bash
+    kubernetes_rootfs="$(dirname $(readlink -f /var/www/ncn-m002/rootfs))"
```

Comment:
Testing this works on the PIT in the next few minutes once redbull is setup for smoke-testing.

Reply:
Had some amendments, works now. Only works with stable tags.